Locale.php

```diff
@@ -85,7 +85,9 @@ public static function getFallback($locale): ?string
 
             array_pop($localeSubTags);
 
-            return locale_compose($localeSubTags);
+            $fallback = locale_compose($localeSubTags);
+
+            return false !== $fallback ? $fallback : null;
```

Comment:
А вот так разве не лучше?

```suggestion
              return locale_compose($localeSubTags) ?: null;
```